pallets/process-validation/src/benchmarking.rs

```diff
@@ -1,25 +1,51 @@
 // ! Benchmarking setup for pallet-template
-
 use super::*;
-
 use frame_benchmarking::{benchmarks, impl_benchmark_test_suite};
-use frame_support::bounded_vec;
+use frame_support::BoundedVec;
 use frame_system::RawOrigin;
 
 #[allow(unused)]
 use crate::Pallet as ProcessValidation;
 
-// TODO implement benchmarking
 benchmarks! {
   create_process {
-  }: _(RawOrigin::Root, T::ProcessIdentifier::default(), bounded_vec![])
+    let i in 1..200;
+
+    let mut program = BoundedVec::<_, _>::with_bounded_capacity(i as usize);
+    for j in 0..i {
+      if j == 0 {
+        program.try_push(BooleanExpressionSymbol::Restriction(Restriction::None)).unwrap();
+      }
+      // add every other loop to have valid postfix notation
+      else if j % 2 == 0 {
+        program.try_push(BooleanExpressionSymbol::Restriction(Restriction::None)).unwrap();
+        program.try_push(BooleanExpressionSymbol::Op(BooleanOperator::And)).unwrap();
+      }
+    }
+
+  }: _(RawOrigin::Root,
+  T::ProcessIdentifier::default(),
+  program.clone())
   verify {
+    // let version = ProcessValidation::<T>::get_version(&T::ProcessIdentifier::default());
+    // let process = ProcessModel::<T>::get(T::ProcessIdentifier::default(), version);
+    // assert_eq!(process.status, ProcessStatus::Enabled)
```

Comment:
not found a way to get this passing correctly. Also tried checking the `i`th version is enabled but the process isn't incrementing its version as expected.

Reply:
any suggestions welcome 🙏

Reply:
hmmm, I'd have to look into it more carefully. We should get this working though before merging